meta/package.yaml

```diff
@@ -1,3 +1,17 @@
+# Copyright (C) 2017 XLAB, Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 name: osv-spark
 title: Spark in OSv
 author: miha
```

Comment:
I suggest you use your full name (name and surname). Having merely "miha" in the source file doesn't satisfy uniqueness factor in your organisation, let alone in the world.